api/src/pcapi/notifications/sms/backends/sendinblue.py

```diff
@@ -68,4 +74,18 @@ def send_transactional_sms(self, recipient: str, content: str) -> bool:
             ),
         )
 
-        return mails.send(recipients=[settings.DEV_EMAIL_ADDRESS], data=mail_content)
+        try:
+            user = users_models.User.query.filter(users_models.User.phoneNumber == recipient).one_or_none()
+        except sa.orm.exc.MultipleResultsFound:
+            logger.error("Several user accounts with the same phone number", extra={"phone_number": recipient})
```

Comment:
il ne faudrait pas utiliser `one()` au lieu de `one_or_none()` justement dans ce cas ? Sinon on ne risque pas d'avoir une `MultipleResultsFound`, si ?

Reply:
https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.one_or_none

Return at most one result or raise an exception.

Returns None if the query selects no rows.

Raises sqlalchemy.orm.exc.MultipleResultsFound if multiple object identities are returned, or if multiple rows are returned for a query that returns only scalar values as opposed to full identity-mapped entities.

Reply:
Au temps pour moi ! Merci pour la doc.